index.html

```diff
@@ -0,0 +1,120 @@
+				<img class="header__logo-image" src="shared/header-logo.png">
+			</div>
+			<div class="header__navigation-container">
+				<nav class="header__navigation-list">
+					<div class="header__navigation-item" id="home">
+						<a href="#home" class="header__navigation-link">Home</a>
+					</div>
+					<div class="header__navigation-item" id="howitworks">
+						<a href="#howitworks" class="header__navigation-link">How it works</a>
+					</div>
+					<div class="header__navigation-item" id="features">
+						<a href="#features" class="header__navigation-link">Features</a>
+					</div>
+					<div class="header__navigation-item" id="partners">
+						<a href="#partners" class="header__navigation-link">Partners</a>
+					</div>
+					<div class="header__navigation-item" id="events">
+						<a href="#events" class="header__navigation-link">Events</a>
+					</div>
+					<div class="header__navigation-item" id="about">
+						<a href="#about" class="header__navigation-link">About us</a>
+					</div>
+				</nav>
+			</div>
+			<div class="header__auth-container">
+				<a href="#" class="header__auth-item">
+						<img class="header__auth-icon" src="shared/sign-in-icon.png"></span>
+						<span class="header__auth-item_small-text">Log in</span>
+				</a>
+				<a href="#" class="header__auth-item">
+					<img class="header__auth-icon" src="shared/avatar-icon.png"></span>
+					<span class="header__auth-item_small-text">Register</span>
+				</a>
+			</div>
+		</header>
+		<main>
+			<article class="landing">
+				<div class="landing__text-container">
+					<div class="landing__text">
+						<div class="landing__text-main">
+							Live Iteraction From Audience
+						</div>
+						<div class="landing__text-common">
+							Q&A, Polls and Slide-sharing for Events
+						</div>
+					</div>
+					<div class="landing__buttons-container">
+						<div class="landing__inner-button-container">
+							<button class="landing__button">
+								Create a event
+							</button>
+						</div>
+						<div class="landing__inner-button-container">
+							<button class="landing__button">
+								Join event
+							</button>
+							<input class="landing__input" type="text" placeholder="type code">
+						</div>
+					</div>
+				</div>
+				<div class="landing__app-container">
+					<img class="landing__app-image" src="shared/app-picture.png">
+				</div>
+			</article>
+			<article class="guide">
+				<div class="guide__title">
+					How it works
+				</div>
+				<div class="guide__steps-container">
+					<div class="guide__step">
+						<div class="guide__image-container guide__image-container_blue-background">
+							<img class="guide__image" src="shared/phone-icon.png">
+						</div>
+						<div class="guide__step-description">
+							Create a FREE account with us
+						</div>
+					</div>
+					<div class="guide__step">
+						<div class="guide__image-container">
+							<img class="guide__image" src="shared/discuss-issue-icon.png">
```

Comment:
please provide `alt = ...` attribute for image, so that user will see some text in case the image wasn't loaded